tests/installation/install_and_reboot.pm

```diff
@@ -48,8 +48,8 @@ sub run {
     my $self = shift;
     # NET isos are slow to install
     my $timeout = 2000;
-    # and encryption makes it even slower
-    $timeout *= 2 if get_var('ENCRYPT');
+    # encryption, LVM and RAID makes it even slower
+    $timeout *= 2 if get_var('ENCRYPT') or get_var('LVM') or get_var('RAID');
```

Comment:
Because of 'or' precedence ambiguity it's not common to use it in if statements and only in "wordy" statements. I would recommend to use || operator here.